charts/ingress-nginx/templates/default-backend-poddisruptionbudget.yaml

```diff
@@ -17,5 +17,6 @@ spec:
       {{- include "ingress-nginx.selectorLabels" . | nindent 6 }}
       app.kubernetes.io/component: default-backend
   minAvailable: {{ .Values.defaultBackend.minAvailable }}
+  unhealthyPodEvictionPolicy: IfHealthyBudget
```

Comment:
Please do not hard-code this. It's fine to just also add a value for the default backend. Consider renaming your PR then. 🙂